draft-ietf-quic-transport.md

```diff
@@ -5437,7 +5449,7 @@ application protocol wishes to prevent the connection from timing out; see
 Receivers send ACK frames (types 0x02 and 0x03) to inform senders of packets
 they have received and processed. The ACK frame contains one or more ACK Ranges.
 ACK Ranges identify acknowledged packets. If the frame type is 0x03, ACK frames
-also contain the sum of QUIC packets with associated ECN marks received on the
+also contain the cumulative count of QUIC packets with associated ECN marks received on the
```

Comment:
(you can't take a sum of QUIC packets; there is no addition operation defined for them.)